packages/ra-ui-materialui/src/AdminUI.tsx

```diff
@@ -18,20 +19,40 @@ export const AdminUI = ({
     loginPage = Login,
     authCallbackPage = AuthCallback,
     notification = Notification,
+    error: errorComponent,
```

Comment:
error should default to `Error`

Reply:
@fzaninotto  not sure to understand. Here, if errorComponent is undefined, his parent (<Error>) will create a default display himself